🔀 chore(package.json): update date-fns dependency to version ^2.30.0

ejcenteno opened the pull request:
Update date-fns dependency to version ^2.30.0 for compatibility and potential bug fixes

The date-fns dependency has been updated to version ^2.30.0 to ensure compatibility with the latest features and bug fixes.

ejcenteno committed:
🔀 chore(package.json): update date-fns dependency to version ^2.30.0 for compatibility and potential bug fixes
The date-fns dependency has been updated to version ^2.30.0 to ensure compatibility with the latest features and bug fixes.

ejcenteno committed:
🐛 fix(date-format.test.js): fix date format in test case to match the expected output
The date format in the test case was incorrect. It was using 'YYYY' instead of 'yyyy' for the year format. This caused the test to fail. The date format has been fixed to 'MMM dd, yyyy' to match the expected output.

ejcenteno committed:
📝 docs(DOC.md): update date-fns version and fix date format examples
📚 chore(DOC.md): update link to date-fns format documentation
The `morph-date-format` function in the DOC.md file has been updated to use `date-fns@v2.30.0` instead of `date-fns@v1.30.1`. This update ensures compatibility with the latest version of `date-fns` and provides access to the latest features

jofftiquez commented on DOC.md:
```diff
@@ -230,25 +230,25 @@ Format any valid date using date-fns' `format()` function.
-const time = this.$morphDateFormat(new Date(), 'hh:mm A');
+const formatted = this.$morphDateFormat(date, 'MMM dd, yyyy');
+const year = this.$morphDateFormat(date, 'yyyy');
+const time = this.$morphDateFormat(new Date(), 'hh:mm a..aa');
```
Is this really the correct format?

ejcenteno replied:
Ow sorry it shout be `a` only.

jofftiquez reviewed (changes requested):
Please see comments. Thanks.

ejcenteno committed:
🔀 chore(age.test.js): update CHESTER_BENNINGTON_AGE constant to reflect the correct age
The constant CHESTER_BENNINGTON_AGE has been updated from 44 to 47 to reflect the correct age of Chester Bennington. This change ensures that the test case accurately represents the expected age.

ejcenteno committed:
🐛 fix(DOC.md): remove unnecessary dots in the time format to fix incorrect time display
The unnecessary dots in the time format have been removed to fix the incorrect time display.

ejcenteno commented:
Here's the result of the unit testing,
![image](https://github.com/jofftiquez/vue-morphling/assets/109321468/06488ce8-57e9-412f-a4f3-ee1418e0d3c8)

jofftiquez commented:
@ejcenteno publish in [v1.2.1](https://www.npmjs.com/package/vue-morphling/v/1.2.1) thanks!
